acceptance/lib/facter/acceptance/base_fact_utils.rb

```diff
@@ -355,7 +355,7 @@ def osx_expected_facts(agent)
             'os.hardware'              => 'x86_64',
             'os.name'                  => 'Darwin',
             'os.macosx.build'          => /\d+[A-Z]\d{2,4}\w?/,
-            'os.macosx.product'        => 'Mac OS X',
+            'os.macosx.product'        => agent['platform'].match(/osx-11-/) ? 'macOS' : 'Mac OS X',
```

Comment:
Usually, `match` is used when you need the matched data. Since you just want to check the platform, `agent['platform'] =~ /osx-11-/ is the better option.